File: iris.py
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt


data = pd.read_csv('iris.csv')

summary = data.describe()

plt.boxplot(data['SepalLength'])

plt.scatter(data['SepalLength'],data['SepalWidth'])

variables = data.iloc[:,0:4]

from sklearn.cluster import KMeans

kmeans = KMeans(n_clusters = 3)

kmeans.fit(variables)

data['clusters'] = kmeans.predict(variables)

from sklearn.metrics import confusion_matrix

cm=confusion_matrix(data['Class'],data['clusters'])

data.to_csv('evaluate.csv')

from sklearn.cluster import KMeans

kmeans = KMeans(n_clusters = 3)

kmeans.fit(variables)

data['clusters'] = kmeans.predict(variables)

wcss = kmeans.inertia_

File: Iris.R
data("iris")
data<-data.frame(iris)
iris.X<-iris[,c(1,2,3,4)]
iris.class<-iris[,"Species"]
library("Hmisc")
describe(iris.X)
boxplot(iris.X)
set.seed(123)
describe(iris.X)
Cor<-cor(as.matrix(iris.X))
ixis.x<-scale(iris.X)

result<- kmeans(iris.X,3)
result$cluster
iris$cluster<- result$cluster

view(iris)
table(iris$Species,iris$cluster)
